Makefile

```diff
@@ -45,20 +44,36 @@ help: Makefile ## Display this help
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "; printf "Usage:\n\n    make \033[36m<target>\033[0m [VARIABLE=value...]\n\nTargets:\n\n"}; {printf "    \033[36m%-30s\033[0m %s\n", $$1, $$2}'
 	@grep -E '^(override )?[a-zA-Z_-]+ \??\+?= .*?## .*$$' $< | sort | awk 'BEGIN {FS = " \\??\\+?= .*?## "; printf "\nVariables:\n\n"}; {gsub(/override /, "", $$1); printf "    \033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
+.PHONY: build-prod-images
+build-prod-images: build-prod-ngf-image build-prod-nginx-image ## Build the NGF and nginx docker images for production
+
+.PHONY: build-prod-images-with-plus
+build-prod-images-with-plus: build-prod-ngf-image build-prod-nginx-plus-image ## Build the NGF and NGINX Plus docker images for production
+
 .PHONY: build-images
 build-images: build-ngf-image build-nginx-image ## Build the NGF and nginx docker images
 
 .PHONY: build-images-with-plus
 build-images-with-plus: build-ngf-image build-nginx-plus-image ## Build the NGF and NGINX Plus docker images
 
+.PHONY: build-prod-ngf-image
+build-prod-ngf-image: TELEMETRY_ENDPOINT=$(PROD_TELEMETRY_ENDPOINT)
+build-prod-ngf-image: build-ngf-image ## Build the NGF docker image for production
```

Comment:
Why this syntax vs a one liner?

Reply:
would a one liner look like this?
```
.PHONY: build-prod-ngf-image
build-prod-ngf-image: TELEMETRY_ENDPOINT=$(PROD_TELEMETRY_ENDPOINT) build-ngf-image ## Build the NGF docker image for production
```

I used this for inspiration https://github.com/nginxinc/nginx-gateway-fabric/blob/8f5af82c3933e24be5d8472ba56927e3925175b9/Makefile#L207-L210

Reply:
I was referring to

```
build-prod-ngf-image: ## Build the NGF docker image for production
   TELEMETRY_ENDPOINT=$(PROD_TELEMETRY_ENDPOINT) build-ngf-image
```

But as I look at it more, it is a bit cleaner with multiple lines. Could probably use that strategy elsewhere.